packages/web-components/src/components/va-statement-of-truth/va-statement-of-truth.tsx

```diff
@@ -161,7 +167,7 @@ export class VaStatementOfTruth {
             value={inputValue}
             message-aria-describedby={inputMessageAriaDescribedby}
             required
-            onInput={this.handleInputChange}
+            onVaTextInputChange={this.handleInputChange}
```

Comment:
I'm not sure that I totally understand why we need two "change" events for `va-text-input` now: `onInput` and `onVaTextInputChange`. Can you help me understand why we couldn't continue to use  the native`onInput` here and avoid having to modify va-text-input?